README.md

```diff
@@ -4,11 +4,25 @@
 
 ## Overview
 
-Lorem ipsum dolor
+This sample application aims to show to show the possibilities of **Voucherify Node.js SDK** and **Voucherify.js** libraries. You can try out this app by clicking Deploy to Heroku button.
 
-## Deploying
+## Setup
+
+It is really simple to setup this app. Only what you need to do is follow the steps listed below:
+
+1. You need a set of *Application Keys* and *Client-side Keys* to connect with **Voucherify Platform**.
+Enter to **[app.voucherify.io](http://app.voucherify.io)** and sign up now!
+
+2. Press this button to create an heroku app
+
+    [![Deploy to Heroku](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
+
+    Wait until the deploy window will be open.
+
+3. After opening the deploy window, please go to the **[Configuration](https://app.voucherify.io/#/app/configuration)** page.
+
+    Generate (if it's need) and copy App Keys from the Configuration page and then paste these keys into proper input fields, in the Deploy Window.
```

Comment:
Generate or copy? 

and paste them

bez ostatniego przecinka.

Wcześniej masz deploy windows od małych liter, zdecyduj się na jeden format.